src/services/folderservices.ts

```diff
@@ -164,14 +166,16 @@ const listAllUnstarredFolders = async ({
   return result;
 };
 
-const renameFolder = async ({
+const updateFolder = async ({
   userId,
   folderName,
+  description,
```

Comment:
about this what if the person wants to just update the name without updating the description or even vice versa?

Reply:
about the delete folder thing .you passed folderid and orgid as params..i think we should validate the params because what if the parms is empty?

Reply:
about the update by allowing null value..if the user doesn't pass anything.nothing gets updated?

Reply:
about the null value, if the user doesn't pass anything, nothing gets changed. This way the name can be changed but not the description and vice versa

Reply:
I just removed required() from the validation for foldername and description. Thats better

Reply:
for the params, if its empty, nothing will be returned or Invalid route

Reply:
for the update function in the service function foldername and description is required though

Reply:
for the validation i feel we can do something like this 
const upadeteFolderValidation = {
  query: joi.object({
    orgId: joi.string().required().messages(validationMessages.orgId),
    foldername: joi.string().required().messages(validationMessages.foldername),
    folderId: joi.string().messages(validationMessages.folderId),
    description: joi.string().messages(validationMessages.folderId),
  }),
};   

but i think this would force us to rethink the service function